editor/src/components/canvas/plugins/style-plugins.ts

```diff
@@ -23,7 +26,7 @@ export const Plugins = {
 } as const
 
 export function getActivePlugin(editorState: EditorState): StylePlugin {
-  if (isTailwindEnabled()) {
+  if (isFeatureEnabled('Tailwind') && isTailwindEnabled()) {
```

Comment:
Should that feature switch be inside `isTailwindEnabled`?

Reply:
good point, I put the feature flag check here to be explicit about the fact that this won't kick in until the FS is removed